server/controllers/userController.js

```diff
@@ -1,107 +1,117 @@
-import dotenv from 'dotenv';
 import jwt from 'jsonwebtoken';
 import moment from 'moment';
 import db from '../models';
 
-dotenv.config({ silent: true });
 
 const secret = process.env.SECRET || 'Happypeopledontkeepsecret';
 const expires = moment().add(1, 'days').valueOf();
 
 
 const UserController = {
-  createNewUser(req, res) {
-    db.users.findOne({ where: { email: req.body.email } })
+  createNewUser(request, response) {
+    const firstName = request.body.firstName;
+    const lastName = request.body.lastName;
+    const email = request.body.email;
+    const userName = request.body.userName;
+    const password = request.body.password;
+    if (!firstName || !lastName || !email || !userName || !password) {
+      return response.status(400).send({
+        message: 'The paramaters are incomplete',
+      });
+    }
+    db.users.findOne({ where: { $or: { email, userName } } })
       .then((userExists) => {
         if (userExists) {
-          return res.status(400).send({
-            message: 'There is a user already existing with this email'
+          return response.status(409).send({
```

Comment:
Status code of 400 would be more appropriate and it would also be consistent with the error codes you have returned for other similar scenarios.